app/components/blacklight/response/facet_group_component.html.erb

```diff
@@ -6,7 +6,7 @@
     <%= collapse_toggle_button(@panel_id) %>
   </div>
 
-  <div id="<%= @panel_id %>" class="facets-collapse d-lg-block collapse accordion">
+  <div id="<%= @panel_id %>" class="<%= body_classes %>">
```

Comment:
I'm a little out of the game. I won't block on this, but do we need a test for this functionality?

Reply:
I've added a test.